ygot/struct_validation_map_test.go

```diff
@@ -1831,6 +1933,7 @@ func TestMergeStructs(t *testing.T) {
 		name    string
 		inA     ValidatedGoStruct
 		inB     ValidatedGoStruct
+		inOpts  []MergeOpt
 		want    ValidatedGoStruct
 		wantErr string
 	}{
```

Comment:
Can we add one test case here that exercises the merge options -- it doesn't need to be exhaustive, but it would be useful to ensure that the implementation passes this through correctly going forward.

Reply:
`TestMergeStructs` already uses the common test cases defined in `mergeStructTests` which has test cases with merge option.